cmd/service-ca-operator/main.go

```diff
@@ -23,6 +23,7 @@ func main() {
 	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 
+	logs.AddFlags(pflag.CommandLine)
```

Comment:
separate commit, fix like this: https://github.com/openshift/cluster-authentication-operator/commit/a2f0d1f994c7d8a8abe5c9beef4b97e02e244608

Reply:
85ccd9a